### PR TITLE
Solve the problem of fetching prediction and make data dim configurable

### LINKED ISSUE #771: Cannot find fetch variable in scope
In inference
```
Traceback (most recent call last):
  File "infer_by_ckpt.py", line 180, in <module>
    infer_from_ckpt(args)
  File "infer_by_ckpt.py", line 165, in infer_from_ckpt
    return_numpy=False)
  File "/home/baidu/liuyibing/paddle_envs/latest/local/lib/python2.7/site-packages/paddle/fluid/executor.py", line 373, in run
    self.executor.run(program.desc, scope, 0, True, True)
paddle.fluid.core.EnforceNotMet: Cannot find fetch variable in scope, fetch_var_name is fc_5.tmp_2 at [/home/baidu/liuyibing/paddle_work/Paddle_tmp/paddle/fluid/operators/fetch_op.cc:40]
PaddlePaddle Call Stacks:
0       0x7f1defb0b037p paddle::platform::EnforceNotMet::EnforceNotMet(std::__exception_ptr::exception_ptr, char const*, int) + 727
1       0x7f1defe93205p paddle::operators::FetchOp::RunImpl(paddle::framework::Scope const&, boost::variant<paddle::platform::CUDAPlace, paddle::platform::CPUPlace, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_, boost::detail::variant::void_> const&) const + 1573
```

### PR DESCRIPTION
**Pull request by kuke**
Fix #771

### CODE REVIEW

File: fluid/DeepASR/infer_by_ckpt.py [hunk 1]
```diff
@@ -32,6 +32,11 @@ def parse_args():
         default=1,
         help='The minimum sequence number of a batch data. '
         '(default: %(default)d)')
+    parser.add_argument(
```
Comment (pkuyym) [inline fluid/DeepASR/infer_by_ckpt.py:35]: @zhxfl Seems this value should be calculated by data reader.

File: fluid/DeepASR/infer_by_ckpt.py [hunk 2]
```diff
@@ -47,6 +52,11 @@ def parse_args():
         type=int,
         default=1024,
         help='Hidden size of lstmp unit. (default: %(default)d)')
+    parser.add_argument(
+        '--class_num',
```
Comment (pkuyym) [inline fluid/DeepASR/infer_by_ckpt.py:56]: Same as above.

File: fluid/DeepASR/model_utils/model.py [hunk 3]
```diff
@@ -20,6 +21,7 @@ def stacked_lstmp_model(hidden_dim,
         label data respectively. And in inference, only `feature` is needed.
 
     Args:
+    frame_dim(int): The frame dimension of feature data.
```
Comment (pkuyym) [inline fluid/DeepASR/model_utils/model.py]: Please align the indent.

Comment (kuke) [reply fluid/DeepASR/model_utils/model.py]: Done

File: fluid/DeepASR/infer_by_ckpt.py (hunk 1 above)
Comment (kuke) [reply fluid/DeepASR/infer_by_ckpt.py:35]: There is a contradiction: After the data reader really begins to read data, it knows the actual dimensions of frame and label data. But before that, the network has been parsed and the right dimensions of feeding data are needed in advance. So the better way is to set this dimension manually.

File: fluid/DeepASR/infer_by_ckpt.py (hunk 2 above)
Comment (kuke) [reply fluid/DeepASR/infer_by_ckpt.py:56]: Explained in last comment.

**Review by pkuyym — APPROVED** [review]
LGTM

**Pull request merged**